README.md

```diff
@@ -51,15 +41,81 @@ SELECT * FROM userlog WHERE ts >= previousTime() AND  ts <= currentTime() ; //ti
-| topic | name for the topic(Kafka Connector property) |
-
-#### Read configuration tuning functionality is in progress.
+
+#### CQL Types Supported
+
+| CQL Type | Schema Type |
+|----------|------------|
+| ASCII | STRING | 
+| VARCHAR | STRING | 
+| TEXT | STRING | 
+| BIGINT | INT64 | 
+| COUNTER | INT64 |
+| BOOLEAN | BOOLEAN |
+| DECIMAL | FLOAT64 |
+| DOUBLE | FLOAT64 |
+| FLOAT | FLOAT32 |
+| TIMESTAMP | TIMESTAMP |
+| VARINT | INT64 |
+
+All the others (BLOB,INET,UUID,TIMEUUID,LIST,SET,MAP,CUSTOM,UDT,TUPLE,SMALLINT,TINYINT,DATE,TIME) are currently NOT supported.
+
+
+## Configuration
+
+#### Sample Source Config
+```
+name=cassandra-source-connector
+connector.class=com.tuplejump.kafka.connect.cassandra.CassandraSource
+tasks.max=1
+
+cassandra.connection.host=10.0.0.1
+cassandra.connection.port=9042
+
+cassandra.source.route.topic1=SELECT * FROM userlog ;
+```
+
+#### Sample Sink Config
+```
+name=cassandra-source-connector
+connector.class=com.tuplejump.kafka.connect.cassandra.CassandraSink
+tasks.max=1
+
+cassandra.connection.host=10.0.0.1
+cassandra.connection.port=9042
+
+cassandra.source.route.topic1=keyspace1.table1
+```
+
+
+| name              | description                | default value  |
+|--------           |----------------------------|-----------------------|
+| | **Connect Properties (for both Source and Sink)** | |
+| name              |  Unique name for the connector. Attempting to register again with the same name will fail.   |                              |
+| connector.class   | The Java class for the connector,  `com.tuplejump.kafka.connect.cassandra.CassandraSource` or `com.tuplejump.kafka.connect.cassandra.CassandraSink` |  |
+| tasks.max         |  The maximum number of tasks that should be created for this connector. The connector may create fewer tasks if it cannot achieve this level of parallelism. | |
+| | **Cassandra Properties (for both Source and Sink)** | |
+| cassandra.connection.host | The host name or IP address to which the Cassandra native transport is bound. | localhost |
+| cassandra.connection.port | Port for native client protocol connections. | 9042 |
+| cassandra.connection.auth.username | Cassandra username | '' |
+| cassandra.connection.auth.password | Cassandra password | '' |
+| cassandra.connection.timeout.ms | Connection timeout duration (in ms) | 8000 |
+| cassandra.connection.read.timeout | Read timeout duration (in ms) | 120000 |
+| cassandra.connection.keep_alive (not yet implemented) | Period of time, in ms, to keep unused connections open. |  |
+| cassandra.connection.reconnect.delay.min | Minimum period of time (in ms) to wait before reconnecting to a dead node. | 1000 |
+| cassandra.connection.reconnect.delay.max | Maximum period of time (in ms) to wait before reconnecting to a dead node. | 60000 |
+| cassandra.connection.compression (broken) | One of the compressor strategies. Can be DeflateCompressor, SnappyCompressor or LZ4Compressor | NONE |
```

Comment:
I found this broken. 
Since we are doing ` ProtocolOptions.Compression.valueOf(s)` in https://github.com/tuplejump/kafka-connect-cassandra/blob/DB-53/src/main/scala/com/tuplejump/kafka/connect/cassandra/CassandraCluster.scala#L213 ,  the possible values should be `snappy` or `lz4` or ``(empty)